backend/schemas/user.py

```
@@ -0,0 +1,19 @@
+    password: str
+    activate: bool
+
+
+class UserCreate(UserBase):
+    pass
+
+
+class UserOut(UserBase):
+    id: int
+
+    class Config:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)